certificate-service/integration-test/src/test/java/se/inera/intyg/certificateservice/integrationtest/fk7211/FK7211ActiveIT.java

```diff
@@ -3481,4 +3483,143 @@ void shallReturn403IfCertificateIsAlreadyRevoked() {
       assertEquals(403, response.getStatusCode().value());
     }
   }
+
+  @Nested
+  @DisplayName("FK7211 - Utökad behörighet vid djupintegration")
+  class AccessLevelsDeepIntegration {
+
+    @Test
+    @DisplayName("FK7211 - Om intyget är utfärdat på en annan enhet inom samma vårdgivare skall det gå att läsa intyget")
+    void shallReturnCertificateIfOnDifferentUnitButSameCareProvider() {
+      final var testCertificates = testabilityApi.addCertificates(
+          defaultTestablilityCertificateRequest(FK7211, VERSION)
+      );
+
+      final var response = api.getCertificate(
+          customGetCertificateRequest()
+              .user(ajlaDoktorDtoBuilder()
+                  .accessScope(AccessScopeTypeDTO.WITHIN_CARE_PROVIDER)
+                  .build())
+              .unit(ALFA_HUDMOTTAGNINGEN_DTO)
+              .build(),
+          certificateId(testCertificates)
+      );
+
+      assertNotNull(
+          certificate(response.getBody()),
+          "Should return certificate when exists!"
+      );
+    }
+
+    @Test
+    @DisplayName("FK7211 - Om intyget är utfärdat på en annan enhet inom samma vårdgivare skall det gå att hämta PDF")
+    void shallReturnPdfIfOnDifferentUnitButSameCareProvider() {
+      final var testCertificates = testabilityApi.addCertificates(
+          defaultTestablilityCertificateRequest(FK7211, VERSION)
+      );
+
+      final var response = api.getCertificatePdf(
+          customGetCertificatePdfRequest()
+              .user(ajlaDoktorDtoBuilder()
+                  .accessScope(AccessScopeTypeDTO.WITHIN_CARE_PROVIDER)
+                  .build())
+              .unit(ALFA_HUDMOTTAGNINGEN_DTO)
+              .build(),
+          certificateId(testCertificates)
+      );
+
+      assertNotNull(
+          pdfData(response.getBody()),
+          "Should return pdf when exists!"
+      );
+    }
+
+    @Test
+    @DisplayName("FK7211 - Om intyget är utfärdat på en annan enhet inom samma vårdgivare skall felkod 403 (FORBIDDEN) returneras vid skickande av intyg")
+    void shallNotAllowToSendOnDifferentUnitButSameCareProvider() {
+      final var testCertificates = testabilityApi.addCertificates(
+          defaultTestablilityCertificateRequest(FK7211, VERSION)
+      );
+
+      final var response = api.sendCertificate(
+          customSendCertificateRequest()
+              .user(ajlaDoktorDtoBuilder()
+                  .accessScope(AccessScopeTypeDTO.WITHIN_CARE_PROVIDER)
+                  .build())
+              .unit(ALFA_HUDMOTTAGNINGEN_DTO)
+              .build(),
+          certificateId(testCertificates)
+      );
+
+      assertEquals(403, response.getStatusCode().value());
+    }
+  }
+
+  @Nested
+  @DisplayName("FK7211 - Utökad behörighet vid SVOD")
```

Comment:
Suggest we include "djupintegration" in the diplayname. Maybe something like: "FK7211 - Utökat behörighet vid djupintegration och SVOD (sjf=true)". Then "FK7211 - Utökad behörighet vid djupintegration" can also add something saying it is without SVOD.